pkg/midstream/write.go

```diff
@@ -102,20 +93,18 @@ func (m *Midstream) mergeKustomization(existing *kustomizetypes.Kustomization) {
 	newResources := findNewStrings(m.Kustomization.Resources, existing.Resources)
 	m.Kustomization.Resources = append(existing.Resources, newResources...)
 
-	m.Kustomization.CommonAnnotations = mergeMaps(m.Kustomization.CommonAnnotations, existing.CommonAnnotations)
-	delete(m.Kustomization.CommonAnnotations, "kots.io/app-slug")
-	delete(m.Kustomization.CommonAnnotations, "kots.io/app-sequence")
-
 	m.Kustomization.CommonLabels = mergeMaps(m.Kustomization.CommonLabels, existing.CommonLabels)
 }
 
 func mergeMaps(new map[string]string, existing map[string]string) map[string]string {
 	merged := existing
-	if merged == nil {
-		merged = make(map[string]string)
-	}
-	for key, value := range existing {
-		merged[key] = value
+	if new != nil {
```

Comment:
why this change?  it is safe to use nil map in a loop

Reply:
oh, i did not know that, will update 👍